_schemaV2.graphql

```diff
@@ -20133,19 +20144,41 @@ union SecondFactorOrErrorsUnion = AppSecondFactor | Errors | SmsSecondFactor
 
 # A piece that can be sold
 interface Sellable {
+  availability: String
   dimensions: dimensions
+  displayLabel: String
+  displayPriceRange: Boolean
   editionOf: String
 
+  # A globally unique ID.
+  id: ID!
+
+  # Price for internal partner display, requires partner access
+  internalDisplayPrice: String
+
+  # A type-specific ID likely used as a database ID.
```

Comment:
This one is interesting - it's an editionSet id? Or the artworkId?

Reply:
Its a "sellable" ID (since it implements the interface) -- which just plucks out the internalID, however it may be